src/layers/medPython/core/medPythonCoreInit.cpp

```diff
@@ -0,0 +1,291 @@
+/*==============================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2021. All rights reserved.
+ See LICENSE.txt for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+==============================================================================*/
+
+// Must be included first (see the header notes for details)
+#include "medPythonCoreAPI.h"
+
+#include "medPythonCoreInit.h"
+
+#include <QDebug>
+#include <QFileInfo>
+#include <QTemporaryDir>
+
+#include <medExternalResources.h>
+
+#include "medPythonCoreUtils.h"
+
+namespace med::python::core
+{
+
+namespace
```

Comment:
is this necessary?

Reply:
For a simple answer: https://google.github.io/styleguide/cppguide.html#Internal_Linkage
(I use namespaces instead of singleton classes, and elements inside unnamed namespaces is sort of the equivalent of private members for a class)